CODE_OF_CONDUCT.md

```diff
@@ -15,4 +15,6 @@ this:
   politely asked to change something. We appreciate any sort of contributions, and don't want a wall
   of rules to get in the way of that.
 
+The uPortal Community is a global community. Because conversations are time-shifted and geo-separated, it is especially important to use appropriate care when communicating. uPortal has a strong tradition of operating through consensus and compromise; flexibility is appreciated and a fair bit of deference is given to the person actually making the contribution. Keeping in mind an emphasis on respect, compassion, and empathy for others helps us avoid or resolve conflicts. When conflicts do arise, we encourage direct communication towards a mutually satisfactory resolution. Ideally we can harness the passion within the community towards effective change and growth. The uPortal Steering Committee acts as a final arbiter for conflict resolution.
```

Comment:
> deference is given to the person actually making the contribution

I'm cautious about the wording here.
At some point we need to be able to say "no" and decline contributions that aren't a good fit for the project.

> we encourage direct communication towards a mutually satisfactory resolution.

What is the encouraged medium for direct communication?

> The uPortal Steering Committee acts as a final arbiter for conflict resolution

How does one contact the steering committee?

Reply:
I am open to another word or phrase to replace 'deference'. The point is, if you are doing the work your voice typically carries more weight than people sitting on the sidelines complaining. Anyone have some suggested wording?

I don't feel we need to prescribe or even suggest a medium for communication.

The uPSC email is given two paragraphs up on the page; do you think it should be repeated?

Reply:
> The uPSC email is given two paragraphs up on the page; do you think it should be repeated?

yes, markdown provides reference links to help reduce duplication.

E.G.

```markdown

In addition to reporting mechanisms under that Policy, participants may at any time contact the [uPortal Steering Committee][] about any matter, and may designate their contact as confidential if they prefer. All contact will be acknowledged. Contacts regarding issues covered by the Welcoming Policy will be handled in a manner consistent with that policy.

...

The uPortal Community is a global community. Because conversations are time-shifted and geo-separated, it is especially important to use appropriate care when communicating. uPortal has a strong tradition of operating through consensus and compromise; flexibility is appreciated and a fair bit of deference is given to the person actually making the contribution. Keeping in mind an emphasis on respect, compassion, and empathy for others helps us avoid or resolve conflicts. When conflicts do arise, we encourage direct communication towards a mutually satisfactory resolution. Ideally we can harness the passion within the community towards effective change and growth. The [uPortal Steering Committee][] acts as a final arbiter for conflict resolution.

[uportal steering committee]: mailto:uportal-steering-committee@apereo.org
```

Reply:
> I am open to another word or phrase to replace 'deference'.

I'm also open to other wording, or removing this sentence.
Since a later section, seems to communicate a similar idea:

> When conflicts do arise, we encourage direct communication towards a mutually satisfactory resolution. Ideally we can harness the passion within the community towards effective change and growth.